lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +52,11 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        if self._punctuation:
+            for char in text:
```

Comment:
consider iterating over punctuation symbols instead, and try to think why it might be better

alternatively, if you find the current way to be the best, I would also like to read your thoughts

Reply:
yeah, it must be faster to iterate over the punctuation instead of the whole text